### PR TITLE
Update hard coded colors to PF tokens and vars

### PR DESCRIPTION
**Pull request by rohitkrai03**
**Fixes**: https://issues.redhat.com/browse/ODC-1166

**Analysis / Root cause**: We have a lot of colors hard coded in our code.

**Solution Description**: Use PF tokens and variables wherever available.

**Screen shots / Gifs for design review**: No design changes
 

**Browser conformance**: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [x] Edge

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/utils.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as _ from 'lodash';
 import { TFunction } from 'i18next';
+import { global_palette_black_400 as globalBlack400 } from '@patternfly/react-tokens';
```
Comment (spadgett) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/utils.tsx:3]: Per @dtaylor113, we should avoid importing tokens this way. See #8777 and https://bugzilla.redhat.com/show_bug.cgi?id=1952111

Comment (christianvogt) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/utils.tsx:3]: Thanks @spadgett I had no idea this was an issue

Comment (rohitkrai03) [reply frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/utils.tsx]: Updated all the imports to use full paths.

File: frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts [hunk 2]
```diff
@@ -22,8 +24,8 @@ window.monaco.editor.defineTheme('console', {
   colors: {
     'editor.background': editorBackground.value,
     'editorGutter.background': '#292e34', // no pf token defined
-    'editorLineNumber.activeForeground': '#fff',
-    'editorLineNumber.foreground': '#f0f0f0',
+    'editorLineNumber.activeForeground': globalBackground100.value,
```
[the remark this answers is not in the document]
Comment (rohitkrai03) [reply frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts:27]: Well it seems PF docs say something and there code says something else.
Screenshot 2021-06-10 at 2 27 08 PM

[the remark this answers is not in the document]
Comment (rohitkrai03) [reply frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts:27]: It seems like the PF var is correct but the react token has wrong value.

[the remark this answers is not in the document]
Comment (rohitkrai03) [reply frontend/packages/console-shared/src/components/editor/yaml-editor-utils.ts]: Updated to use a different token.

**Review by rottencandy — APPROVED** [review]
/lgtm

**Pull request merged**